src/components/KYCWall/BaseKYCWall.js

```diff
@@ -97,7 +96,7 @@ class KYCWall extends React.Component {
         });
 
         // Check to see if user has a valid payment method on file and display the add payment popover if they don't
-        if (!PaymentUtils.hasExpensifyPaymentMethod(this.props.cardList, this.props.bankAccountList)) {
+        if (!lodashGet(this.props.userWallet, 'hasPaymentMethod', false)) {
```

Comment:
I don't think `hasPaymentMethod` exists on the userWallet prop yet. Is it being set somewhere? Or do we mean to check if the user's wallet has a `walletLinkedAccountID`?